src/Transports.AspNetCore/GraphQLHttpMiddlewareOptions.cs

```diff
@@ -95,7 +97,8 @@ public class GraphQLHttpMiddlewareOptions : IAuthorizationOptions
     public GraphQLWebSocketOptions WebSockets { get; set; } = new();
 
     /// <summary>
-    /// The Content-Type to use for GraphQL responses
+    /// The Content-Type to use for GraphQL responses, if it matches the 'Accept'
```

Comment:
I read it several times and this comment still seems strange to me. I don't understand the meaning of _matches_.

Reply:
I mean in generally I understand what you want to say but from the point of view of some "external" developer it may look strange.

Reply:
I do the best I can.  Sometimes I'm tongue tied.  Here it seemed to make sense to me, but I get it.  Feel free to suggest an improvement.